run_linux.go

```diff
@@ -1203,6 +1203,13 @@ func runCopyStdio(stdio *sync.WaitGroup, copyPipes bool, stdioPipe [][]int, copy
 	runCopyStdioPassData(copyPipes, stdioPipe, finishCopy, relayMap, relayBuffer, readDesc, writeDesc)
 }
 
+func canRetry(err error) bool {
+	if errno, isErrno := err.(syscall.Errno); isErrno {
```

Comment:
Should we use an errors.Cause for this?

Reply:
the error is coming from `unix.Read` and I've verified locally it works.  I can make it safer with errors.Cause if you'd like